### PR TITLE
Feature/support 484 ad logon logging

### PR DESCRIPTION
**Pull request by turegjorup**
* Updated Symfony, EasyAdmin and dependencies
* Added logging for AD logins for debugging
* Added error page for AD login errors

Screenshot 2022-06-09 at 15 20 55

### CODE REVIEW

**Comment by turegjorup** [comment]
Not sure why "Review JS / PHPCS" get's run - it's a PHP check under JS review!

File: .env [hunk 1]
```diff
@@ -40,7 +40,7 @@ DATABASE_URL=mysql://db:db@mariadb:3306/db
 CONFIGURATION_URL=APP_CONFIGURATION_URL
 CLIENT_ID=APP_CLIENT_ID
 CLIENT_SECRET=APP_CLIENT_SECRET
-CALLBACK_URI=APP_CALLBACK_URI
+REDIRECT_URI=APP_REDIRECT_URI
```
Comment (sinejespersen) [inline .env:43]: this is mentioned in the readme, so should be updated

Comment (sinejespersen) [reply .env:43]: updated in the readme that is

**Review by sinejespersen — APPROVED** [review]
Readme update

File: .env (hunk 1 above)
Comment (turegjorup) [reply .env:43]: Fixed

File: templates/box_error/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                        <dd class="col-sm-10">{{ exceptionMessage ?? 'Unknown' }}</dd>
+                    </dl>
+                    <hr>
+                    <p class="mb-0">{{ 'now' | date('Y-m-d\TH:i:sO') }} - {{ 'The error has been logged for debugging.' | trans }}</p>
+                </div>
+            </div>
+            <div class="container-md pt-4">
+                <p class="text-center">
+                    {% if boxUrl %}
+                        <a class="btn btn-outline-dark" href="{{ boxUrl }}" role="button">◀ {{ 'Return to box' | trans }}</a>
```
Comment (tuj) [inline templates/box_error/index.html.twig:32]: We probably want the texts translated to danish.

**Review by tuj — APPROVED** [review]

**Pull request merged**